po/de.po

```diff
@@ -144,31 +144,31 @@ msgstr "Repositorium"
 
 #: src/config.rs:983
 msgid "  failed to sync {}"
-msgstr " {} konnte nicht synchronisiert werden"
+msgstr " {} kann nicht synchronisiert werden"
 
 #: src/devel.rs:129 src/download.rs:179
 msgid "Querying AUR..."
-msgstr "AUR wird abgefragt…"
+msgstr "AUR abfragen…"
 
 #: src/devel.rs:142 src/devel.rs:166 src/install.rs:694
 msgid "failed to parse srcinfo for '{}'"
-msgstr "srcinfo für ‚{}‘ konnte nicht geparsed werden"
+msgstr "kann srcinfo für ‚{}‘ nicht parsen"
 
 #: src/devel.rs:184
 msgid "Looking for devel repos..."
 msgstr "Suche nach devel Repos…"
 
 #: src/devel.rs:193
 msgid "failed to save devel info"
-msgstr "devel Informationen konnten nicht gespeichert werden"
+msgstr "kann devel Informationen nicht speichern"
 
 #: src/devel.rs:382
 msgid "package does not have an update"
-msgstr "für das Paket gibt es kein Update"
+msgstr "Paket ist in der neuesten Version"
```

Comment:
Imo this is wrong. "Package X is the most current version" has other implications than "Package X does not have an update", especially for AUR packages.